src/components/Network.js

```diff
@@ -149,14 +173,14 @@ class Network extends React.Component {
               Uptime: {dateFormatter.uptimeFormat(conn.uptime)}<br />
               Version: {conn.app_version}<br />
               Address: {conn.address}<br />
-              Entrypoints: {peer.entrypoints.join(", ")}<br />
+              Entrypoints: { /* peer.entrypoints.join(", ") */ }<br />
```

Comment:
Why did you comment it out? I think we should show the peer's entrypoints.

Reply:
Add in this issue to add back the entrypoints on explorer when it's added there

Reply:
Done

Reply:
I don't get it. Do we filter out the full-node's response? Anyway, I think we should make this code work when the information is correctly returned instead of this commenting this out. Maybe the API Client can fix it putting an empty list there is nothing is return.

Reply:
I agree, changed to work with or without entrypoints coming

Reply:
Great. Thanks!